addons/mail/static/src/scss/discuss.scss

```diff
@@ -327,9 +327,18 @@
         .o_preview_title {
             display: flex;
             .o_preview_name {
-                flex: 0 1 auto;
+                flex: 1 0 auto;
                 @include o-text-overflow;
             }
+            .o_mail_activity_action_buttons {
+                display: flex;
+                flex: 0 1 auto;
+                flex-flow: row wrap;
```

Comment:
`flex-flow: reverse-row wrap;` (so that the icons are ordered from right to left, similarly to systray items themselves)

Reply:
Yup, should be `row-reverse`